holmes/config.py

```diff
@@ -175,6 +200,11 @@ def load_from_file(cls, config_file: Optional[Path], **kwargs) -> "Config":
         Returns:
             Config instance with merged settings
         """
+        # Import Transformer class to resolve forward reference
+
+        # Rebuild the model to resolve forward references
+        cls.model_rebuild()
```

Comment:
Can you please explain why we need this?

Reply:
this needs to be cleaned up together with global transfomer. thanks for catching.